src/hooks/useSubaccount.tsx

```diff
@@ -436,6 +436,15 @@ export const useSubaccountContext = ({ localDydxWallet }: { localDydxWallet?: Lo
           onSuccess?.(data);
         } else {
           onError?.(data, { errorStringKey: parsingError?.stringKey });
+
+          const { placeOrderPayloads } = data || {};
+          placeOrderPayloads &&
+            placeOrderPayloads.toString() != '[]' &&
```

Comment:
do we have to cast it to a string? can we check that it's empty in other ways?

Reply:
In Abacus it needs to be typed to `kollections.List` in order for `toArray` to work.

ex. from `abacusjs.d.ts`
`get line(): Nullable<kollections.List<number>>;`
whereas the `placeOrderPayloads` is
`get placeOrderPayloads(): any/* kotlin.collections.List<exchange.dydx.abacus.state.manager.HumanReadablePlaceOrderPayload> */;`

may need to bump abacus to correct the type.

Reply:
ohhh that is 10/10 knowledge - ty will try this out!

Reply:
hmm so it all compiles fine, but I do get the warning:

```
Exported declaration uses non-exportable parameter type: IList<HumanReadablePlaceOrderPayload> /* = List<HumanReadablePlaceOrderPayload> */
```

should I be worried about that 🤔